packages/contract/README.md

```diff
@@ -16,76 +16,58 @@
 pnpm add @amqp-contract/contract
 ```
 
-## Usage
+## Quick Start
```

Comment:
you also have to update website documentation

and it would be greate to update sample too